agilesix/diffusion-marketplace
# DM-2356: Add created practices search to facility show

### JIRA issue link
[DM-2356](https://agile6.atlassian.net/browse/DM-2356)

## Description - what does this code do?
This PR adds the created practices search for a facility page.

## Testing done - how did you test it/steps on how can another person can test it 
**Prereq**
1. Set up a facility so that it has 3 or fewer practices that were created at that facility
2. Ensure those practices have some adoptions

**How to Test**
1. Go to the originating facility page of the practices you modified above
2. Ensure the defaults show the following
- 3 results count
- Default to A to Z sort
- No search query
- No categories selected
- No load more button
3. Ensure the sorting works as expected
4. Search for a category name or related term and ensure you see the expected practices and the result count is correct
5. Search for a term in the practices' name, tagline, summary, overview problem, overview solution, overview results, diffusion status (e.g. emerging, replicating, etc), category name, category related term, facility name where practice was adopted, other facility names of originating location
6. Ensure you can filter by categories
7. Ensure when the results are loading there is a loading spinner
8. Ensure you see a no results message when there are no results
9. Repeat 3-8 after adding more practices to the facility so that you see a load more button

## Screenshots, Gifs, Videos from application (if applicable)


## Link to mock-ups/mock ups (image file if you have it) (if applicable)
[Figma](https://www.figma.com/file/fNXjZzLUS39bKmjdfDWyxy/My-VISN%2FVAMC?node-id=2403%3A4050)

## Acceptance criteria
- [X] VAMC has search for practices created, keyword, and category
- [X] 3 to start then load more
- [X] Sort dropdown (most adoptions, most recently created, and A to Z) 

## Definition of done
